test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -417,27 +417,27 @@ static void ipsec_pkt_auth_err_set(odp_packet_t pkt)
 	odp_packet_copy_from_mem(pkt, len - sizeof(data), sizeof(data), &data);
 }
 
-static void ipsec_check_out_in_one(const ipsec_test_part *part,
+static void ipsec_check_out_in_one(const ipsec_test_part *part_out,
+				   ipsec_test_part part_in,
```

Comment:
Why did you choose to pass by value here? Why not pass the pointer? No strong opinion here. Just trying to understand the rationale.

Reply:
Maybe I just did not want to change the existing lines that referred to part_in. But it is a bit silly to pass it by value when part_out is passed by reference. I'll change it in the next version.